backend/census_historical_migration/sac_general_lib/auditee_certification.py

```diff
@@ -0,0 +1,45 @@
+# import audit.validators
+from census_historical_migration.workbooklib.census_models.census import (
+    CensusGen22 as Gen,
+)
+from census_historical_migration.base_field_maps import FormFieldMap, FormFieldInDissem
+from census_historical_migration.sac_general_lib.utils import (
+    _create_json_from_db_object,
+)
+
+# FIXME: Do we need this ?
+# auditee_certification_mappings = [
+#     FormFieldMap("has_no_PII", None, FormFieldInDissem, False, bool),
+#     FormFieldMap("has_no_BII", None, FormFieldInDissem, False, bool),
+#     FormFieldMap("meets_2CFR_specifications", None, FormFieldInDissem, False, bool),
+#     FormFieldMap("is_2CFR_compliant", None, FormFieldInDissem, False, bool),
+#     FormFieldMap("is_complete_and_accurate", None, FormFieldInDissem, False, bool),
+#     FormFieldMap("has_engaged_auditor", None, FormFieldInDissem, False, bool),
+#     FormFieldMap("is_issued_and_signed", None, FormFieldInDissem, False, bool),
+#     FormFieldMap("is_FAC_releasable", None, FormFieldInDissem, False, bool),
+# ]
+
+auditee_signature_mappings = [
+    FormFieldMap("auditee_name", "auditeename", FormFieldInDissem, None, str),
+    FormFieldMap("auditee_title", "auditeetitle", FormFieldInDissem, None, str),
+    # FIXME: Do we need this ?

```

Comment:
I don't think we disseminate it. So, possibly/probably not. I'm working from memory, though.

Reply:
We don't disseminate, I simply added this comment so that we can make a group decision.

Reply:
I decided to keep the field so that we don't run into issue with the validator.